source/_includes/header.html

```diff
@@ -1,35 +1,48 @@
-<div class="FlexDiv Header_Container">
-    <div>
-        <img src="" alt="Skolio Icon">
-    </div>
-    <div>
-        <nav>
-            {% for item in site.data.header_navigation %}
-            {% if item.subitems %}
-            <div class="dropdown">
-            {% endif %}
-                <a class="dropbtn"
-                   {% if item.url %}
-                   href="{{ item.url }}"
-                   {% endif %}
-                >
-                    {{ item.title }}
-                </a>
+<div class="Header_Container">
+    <div class="Header_Background"></div>
+    <div class="container FlexDiv">
+        <div class="Header_BrandContainer">
+            <img src="assets/images/skolio-logo.svg" alt="Skolio Icon">
+        </div>
```

Comment:
The brand icon usually doubles as a link back to the home page. To conform with user expectations, this should also include a link back to /

Reply:
Ah, right. I will add that right away, thanks